### ISSUE TITLE
parked domain: deadhomersociety.com

### ISSUE THREAD

**Issue by ghost** [nextdns/metadata#1254]
a well-known blog ([archive](https://web.archive.org/web/20210210234950/https://deadhomersociety.com/)), now redirects somewhere else entirely

domain: `deadhomersociety.com`

**Comment by PeterDaveHello** [comment]
No longer valid now?

**Comment by ghost** [comment]
> No longer valid now?

I don't think so, [archive.org](https://web.archive.org/web/20230000000000*/deadhomersociety.com) confirms the redirects at the time of writing (September 5, 2023):

>Loading...
>
> `https://deadhomersociety.com/` |
	    17:15:15 September 05, 2023
>
>Got an HTTP 301 response at crawl time
>
>Redirecting to...
>
> `https://h30630.www3.hp.com/slot88/`

**Comment by PeterDaveHello** [comment]
Interesting, I didn't see that 301 redirect, but I do see screen like this:

![image](https://github.com/nextdns/metadata/assets/3691490/5e552260-5c7d-463a-82e0-6b4ddb77aa91)
